pkg/config/signer_test.go

```diff
@@ -86,16 +86,18 @@ func TestNewSigner(t *testing.T) {
 			spec:        "empty mspID",
 			publicKey:   []byte(publicKey),
 			privateKey:  []byte(privateKey),
```

Comment:
See above comment and all other places in this test where it's casted

Reply:
done